appscale/tools/agents/euca_agent.py

```diff
@@ -54,7 +54,7 @@ def open_connection(self, parameters):
       self.handle_failure('Unknown scheme in EC2_URL: ' + result.scheme)
       return None
 
-    if parameters['IS_VERBOSE']:
+    if parameters.get('IS_VERBOSE', None):
```

Comment:
Very minor Python style nitpick, but `None` is the default value anyways for `.get`, so `parameters.get('IS_VERBOSE')` works here.

`parameters.get('IS_VERBOSE', False)` would also be fine since an explicit boolean is a bit more readable in the context of an "if" statement.